polyfill/intersection-observer.js

```diff
@@ -863,6 +863,30 @@ function getEmptyRect() {
 }
 
 
+/**
+ * Ensure that the result has all of the necessary fields of the DOMRect.
+ * Specifically this ensures that `x` and `y` fields are set.
+ *
+ * @param {?DOMRect} rect
+ * @return {?DOMRect}
+ */
+function fixDOMRect(rect) {
```

Comment:
Maybe `validateDOMRect()`, since it's not necessarily fixing anything, right?

Reply:
Changed to ensureDOMRect.